LoggerConfig/LoggerImplementation/AutoLog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Discord;
+using Serilog.Events;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig.LoggerImplementation;
```

Comment:
This class doesn't really have anything to do with configuration, so I think changing the namespace both here and in `SerilogConfiguration.cs` to something like `QuickEdit.Logger` or `QuickEdit.Logging` *could* be better? I'm not sure though